app/models/landable/access_token.rb

```diff
@@ -2,19 +2,41 @@ module Landable
   class AccessToken < ActiveRecord::Base
     include Landable::TableName
 
+    # Maximum token age, in hours
+    MAX_AGE = (Landable.configuration['ldap'][:access_token_max_age] || 8).hours
+
     belongs_to :author
     validates_presence_of :author_id
     validates_presence_of :expires_at
+    validates_presence_of :permissions
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.